cmd/meroxa/root/apps/utils.go

```diff
@@ -0,0 +1,214 @@
+package apps
```

Comment:
There is no:
- `app.json` (one day, we will probably have `app.yaml`)
- the need for using git.

Reply:
I'm removing this for now until we will add app.yaml support!